addons/mail/tests/test_discuss_controller.py

```diff
@@ -342,3 +344,31 @@ def test_mail_partner_from_email_authenticated(self):
             self.env["res.partner"].search_count([('email', '=', "john2@test.be")]),
             "'mail/message/post' does not create another user if there's already a user with matching email",
         )
+
+    def test_mail_cache_control_header(self):
+        testuser = self.env['res.users'].create({
+            'email': 'testuser@testuser.com',
+            'groups_id': [Command.set([self.ref('base.group_system')])],
```

Comment:
It would be nice to test with non-admin, or lowest possible access generally speaking, internal user, portal, or even public.